### ISSUE TITLE
The meaning of your code

### ISSUE THREAD

**Issue by jampeg** [weiyinwei/MMGCN#13]
Hello, 
I'm tryng to your MMGCN.
Why did you subtract 'self.num_user' on the line 205 in the Model_MMGCN.py, and plus 'self.num_user' on the line 209 in the Model_MMGCN.py?
I cannot understand it.

From Model_MMGCN.py
line 205: `col = torch.LongTensor(list(col))-self.num_user`
line 209: `all_index_of_rank_list = torch.cat((all_index_of_rank_list, index_of_rank_list.cpu()+self.num_user), dim=0)`

Best regards.

**Comment by weiyinwei** [comment]
The size of score_matrix is num_user*num_col. But the range of the variable 'col' is [num_user, num_user+num_col].
